src/api/app/components/build_status_for_architecture_component.html.haml

```diff
@@ -0,0 +1,31 @@
+:ruby
+  result_code = result.code
+  result_help = helpers.repository_info(result.state) if result.is_repository_in_db
+  result_help ||= 'This result is outdated'
+  help = "<p><strong>Package build (#{result_code}):</strong> #{Buildresult.status_description(result_code)}</p>" \
+         "<p><strong>Repository status (#{result.state}): </strong>#{result_help}</p>"
+.result.p-3.my-3.my-sm-4.mx-3{ class: "#{helpers.status_border_color(result_code)}" }
+  .d-flex.flex-row.justify-content-between.align-items-baseline
+    .architecture
+      %span.font-weight-bold= result.architecture
+    %i.fa.fa-lg.fa-question-circle.text-info{ data: { placement: 'top',
+                                                      toggle: 'popover',
+                                                      html: 'true',
+                                                      content: help } }
+
+  .d-flex.flex-row.flex-wrap.justify-content-between.mt-3
+    .build-status.mr-3
+      - if result_code.in?(['-', 'unresolvable', 'blocked', 'excluded', 'scheduled'])
+        %i{ class: "fa #{helpers.build_status_icon(result_code)} text-gray-500" }
+      - else
+        = link_to(package_live_build_log_path(project: project.to_s, package: package,
+            repository: result.repository, arch: result.architecture), rel: 'nofollow') do
+          %i{ class: "fa #{helpers.build_status_icon(result_code)} text-gray-500" }
+      %span
+        -# TODO: rework arch_repo_table_cell if we finally remove the colors from the status text.
+        = result_code
+        =# helpers.arch_repo_table_cell(result.repository, result.architecture, package, 'code' => result_code, 'details' => result.details)
+    .repository-status
+      = helpers.repository_status_icon(status: result.state)
+      %span
+        = result.state
```

Comment:
What about dropping this text and moving it as an icon tooltip text only?
The status is already reported in the help box and maybe it is not so relevant to display at the first glance. The icon giving an initial clue about its state could be enough.
By that it would get less verbose and every "box" can be shrunk a bit.

Reply:
In such case, I'll move the icon in front of the architecture:

![Screenshot 2022-11-11 at 16-43-37 Open Build Service](https://user-images.githubusercontent.com/2581944/201375344-f44f5f80-40b5-4a41-a80d-9e530a33cc6a.png)

Otherwise it will look like this:
![Screenshot 2022-11-11 at 16-39-49 Open Build Service](https://user-images.githubusercontent.com/2581944/201375500-39cdf6bc-e377-4de6-a159-680606e495e4.png)

Reply:
... what makes me think that we don't need two lines for that...